Assets/MRTK/Providers/OpenXR/Scripts/MicrosoftArticulatedHand.cs

```diff
@@ -229,6 +296,31 @@ private void UpdateHandData(InputDevice inputDevice)
                 handMeshProvider?.UpdateHandMesh();
                 handJointProvider?.UpdateHandJoints(inputDevice, unityJointPoses);
                 handDefinition?.UpdateHandJoints(unityJointPoses);
+
+                // Note: After some testing, it seems when moving your hand fast, Oculus's pinch estimation data gets frozen, which leads to stuck pinches.
```

Comment:
Should this be removed, or will it be revisited soon-ish?

Reply:
Removed because it was an ovr specific change.